benchmark/browser/scenarios/sx-prop-box-material-ui/index.js

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import { ThemeProvider, createMuiTheme } from '@material-ui/core/styles';
+import Box from '@material-ui/core/Box';
+import { logReactMetrics } from '../utils';
+
+const theme = createMuiTheme();
+
+export default function BasicStyledComponents() {
+    return (
+      <React.Profiler id="sx-prop-box-material-ui" onRender={logReactMetrics}>
+        {new Array(1000).fill().map(() => (
+          <ThemeProvider theme={theme}>
```

Comment:
Do we need the theme provider?

Reply:
Do we need the theme provider inside the loop?

Reply:
I've added it just so it would be heave as the theme-ui example.. We don't really need it here. For `theme-ui` we can move it outside of the loop